envs/windows-pinned.yaml

```diff
@@ -1,501 +1,501 @@
-# SPDX-FileCopyrightText: PyPSA-Earth and PyPSA-Eur
-#
-# SPDX-License-Identifier: CC0-1.0

```

Comment:
The pinned-environment files are being generated in an automated way, and it's generally better to avoid changing them manually. Instead, it makes absolute sense to fix the script used to generate them: [this one](https://github.com/pypsa-meets-earth/pypsa-earth/blob/main/.github/workflows/update-pinned-env.yml)

Reply:
I am not sure why it was being flagged. But only `windows-pinned.yaml` was affected which is strange.

Reply:
The pre-commit issue is complaining about the return character ('\n') that in windows is not the same as in linux:
The error is:
envs/windows-pinned.yaml:1:52: [error] wrong new line character: expected \n (new-lines)

When a new PR is merged, the failed issue will disappear. A long-term solution is to revise the ending character for all files (other yamls suffer the same issue) or disable in pre-commit this check

Reply:
The difference for example may lie here:
https://github.com/pypsa-meets-earth/pypsa-earth/blob/1ac40ac4a458b0f2ba6e97effe53c2c4f2db8ff7/.github/workflows/update-pinned-env.yml#L54
It is probably safer to disable this check for selected yamls

Reply:
To ignore the check on the pinned, we could add an option like this one:
https://github.com/pypsa-meets-earth/pypsa-earth/blob/1ac40ac4a458b0f2ba6e97effe53c2c4f2db8ff7/.pre-commit-config.yaml#L76
for the rule that is triggering the issue, that is likely this one:
https://github.com/pypsa-meets-earth/pypsa-earth/blob/1ac40ac4a458b0f2ba6e97effe53c2c4f2db8ff7/.pre-commit-config.yaml#L80

Reply:
I am going for the option to disable the check for pinned yaml file